### PR TITLE
fix(connect-popup): loader tuning

### LINKED ISSUE #5644: Infinite spinner on Connect for Firefox users with Total Cookie Protection
**Describe the bug**
Using [MyEtherWallet](https://www.myetherwallet.com/wallet/access/hardware?type=overview) on Firefox with Total Cookie Protection results in a continuous loading on connect.trezor.io. 

**Info:**
-   Browser: Firefox 101.0.1
-   OS: OSX 12.2.1
-   Bridge: 2.0.31
-   Firmware Version: TT, 2.5.1

**How to reproduce**
Setup Firefox Total Cookie Protection. Settings > Privacy and Security

 
Screenshot 2022-06-17 at 16 42 12

1. Go to https://www.myetherwallet.com/wallet/access/hardware?type=overview
2. Click Trezor
3. Note the loading spinner

**Screenshots**
Make exemption for particular problem site:
Screenshot 2022-06-17 at 12 21 15

Console error
> Partitioned cookie or storage access was provided to “https://connect.trezor.io/8/iframe.html?version=8.2.8&manifest=IntcImVtYWlsXCI6XCJkZXZAbXlldGhlcndhbGxldC5jb21cIixcImFwcFVybFwiOlwiaHR0cHM6Ly93d3cubXlldGhlcndhbGxldC5jb21cIn0i” because it is loaded in the third-party context and dynamic state partitioning is enabled.

Screenshot 2022-06-17 at 16 52 55

**Additional context**
The feature is only available to 'easy access' users and is opt in: https://support.mozilla.org/en-US/kb/total-cookie-protection#w_how-do-i-know-if-i-have-early-access

Connect v6 wallet does behave like v8 and thus works in Firefox. Example wallet: https://wallet.xinfin.network/#/access-my-wallet

Suggestion from @mroz22 is to not auto-close the popup in case of error and print it. Though in this case, the popup does not close.

### PR DESCRIPTION
**Pull request by mroz22**
- ~~This should fix connect-popup in firefox. The problem is that broadcast channel is available, and popup in firefox tries to use it, but messages from popup (eg. POPUP.HANDSHAKE) never reach iframe or maybe other way round.
So, if there is already a fallback in place but it becomes active only if BroadcastChannel is undefined. Now popup/iframe will resort to the fallbackish style also in case when initial handshake does not happen within 2 seconds.~~ done in https://github.com/trezor/trezor-suite/pull/8436
- added some context messages below loader component. should anything get stuck, we will at least know how far the process went.

### Testing tips:
- you need to have iframe and "host" on different origins. test it wherever you want, it could even be current production, but point it to this branch using trezor-connect-src, for example https://connect.trezor.io/9/?trezor-connect-src=http://localhost:8088/#/method/getPublicKey

cc @Hannsek 

resolve https://github.com/trezor/trezor-suite/issues/5644

### CODE REVIEW

**Comment by mroz22** [comment]
- [ ] @trezor/blockchain-link
- [ ] @trezor/blockchain-link-utils
- [ ] @trezor/blockchain-link-types
- [ ] @trezor/type-utils
- [ ] @trezor/connect-analytics
- [ ] @trezor/analytics
- [ ] @trezor/connect-common
- [ ] @trezor/env-utils
- [ ] @trezor/transport
- [ ] @trezor/utxo-lib
- [ ] @trezor/trezor-user-env-link
- [ ] @trezor/utils

File: packages/connect-ui/src/index.tsx [hunk 1]
```diff
@@ -72,8 +72,14 @@ export const ConnectUI = ({ postMessage, clearLegacyView }: ConnectUIProps) => {
         } else {
             // messages[0] could be null. in that case, legacy view is rendered
             switch (messages[0]?.type) {
+                case 'waiting-for-iframe-init':
+                    component = <Loader message="waiting for iframe to load" />;
+                    break;
+                case 'waiting-for-iframe-handshake':
+                    component = <Loader message="waiting for handshake from iframe" />;
```
Comment (karliatto) [inline packages/connect-ui/src/index.tsx]: Nit: messages seam too technical for End user facing UI. Probably final user does not need to know what `iframe` is.

**Review by karliatto — APPROVED** [review]
Looks good to me.

File: packages/connect-ui/src/index.tsx (hunk 1 above)
Comment (mroz22) [reply packages/connect-ui/src/index.tsx]: true, I'll change it to something more readable.

**Pull request merged**